server/re/src/msi_genquery2.cpp

```diff
@@ -0,0 +1,315 @@
+#include "irods/msi_genquery2.hpp"
+
+#include "irods/irods_at_scope_exit.hpp"
+#include "irods/rs_genquery2.hpp"
+#include "irods/msParam.h"
+#include "irods/irods_re_structs.hpp"
+#include "irods/irods_logger.hpp"
+#include "irods/irods_state_table.h"
+#include "irods/rodsError.h"
+#include "irods/rodsErrorTable.h"
+#include "irods/msi_preconditions.hpp"
+#include "irods/irods_exception.hpp"
+
+#include <fmt/format.h>
+#include <boost/any.hpp>
+#include <nlohmann/json.hpp>
+
+#include <algorithm>
+#include <cstdint>
+#include <cstdlib>
+#include <cstring>
+#include <iterator>
+#include <map>
+#include <string>
+
+namespace
+{
+    using log_msi = irods::experimental::log::microservice;
```

Comment:
Should we use the new genquery2 category?

Reply:
No. That category is specific to the parser implementation. If I'm using that category outside of the parser, then please highlight where I'm doing that so I can change it.

Reply:
Okay, makes sense. I didn't notice any instances of it outside of the parser itself.